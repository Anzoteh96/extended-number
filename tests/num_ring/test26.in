612607899466187000525816856134125936243053806025762517779424572628425790235314
142058879530382016689790105127261689070873347642393832799226507904198010515766
-669032821273413441442037125996885373607960296686309610396903228594904393686506
