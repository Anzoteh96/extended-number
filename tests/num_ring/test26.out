Enter the number base b:
Enter two numbers n and m:
b is 612607899466187000525816856134125936243053806025762517779424572628425790235314
n mod b is 142058879530382016689790105127261689070873347642393832799226507904198010515766
m mod b is 556182977658960559609596586271366498878147315365215425161945916661947186784122
n + m is 85633957723155575773569835264502251705966856981846740181747851937719407064574
n - m is 198483801337608457606010374990021126435779838302940925416705163870676613966958
n * m is 178623425159095142160204869944936568363496317670580482240387971364631130678042
n ** m is 43997471306901904967796181694434272173897739832033457915977307984850842083380
